libs/designsystem/src/lib/components/button/button.component.integration.spec.ts

```diff
@@ -589,3 +590,29 @@ describe('ButtonComponent configured with text and icon', () => {
     });
   });
 });
+
+describe('ButtonComponent using an ngIf directive', () => {
```

Comment:
Please align naming:
```suggestion
describe('ButtonComponent configured with text and icon using an ngIf directive', () => {
```

Reply:
Should be resolved by new commits 😊